### DISCUSSION TITLE
how to remove the Square icone?

### DISCUSSION THREAD

**Discussion by oswaldB** [Budibase/budibase#1941] [Help] [answered]
![image](https://user-images.githubusercontent.com/3023526/124762266-9ec0fb80-df2a-11eb-82e0-72035285dec5.png)

Hello how I can remove this?

Thank you

**Comment by aptkingston** [comment, marked as answer]
Hi @oswaldB. You can't remove that icon, but it's an extremely important icon as it takes users back to the portal home page. You need to use this icon to log out, or change what app you're using.

**Comment by oswaldB** [comment]
Hello,

Yes but if I have only one app.... ?

On Wed, 7 Jul 2021 at 13:57, Andrew Kingston ***@***.***>
wrote:

> Hi @oswaldB <https://github.com/oswaldB>. You can't remove that icon, but
> it's an extremely important icon as it takes users back to the portal home
> page. You need to use this icon to log out, or change what app you're using.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Budibase/budibase/discussions/1941#discussioncomment-974747>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAXCFJV56QD4OXUJ5CL2G53TWRFKJANCNFSM476TMZ5Q>
> .
>


-- 
Oswald BERNARD
00 44 7861004200

**Reply by aptkingston** [reply]
Even if you have one app, you still need to go back to the portal home page to log out, change your password or change your name.
